rajapinnat/logmaster/resend_logmaster.php

```diff
@@ -47,6 +47,8 @@
       $palautus = logmaster_send_file($filename);
 
       if ($palautus == 0) {
+        logmaster_sent_timestamp($laskurow['tunnus']);
```

Comment:
Ei päivitetä täällä timestamppiä. Resend on sallittu vain, jos tilaus on kerran lähetetty (timestamp not null). Muuten tilaus menee väärään tilaan.